internal/server/grants_test.go

```diff
@@ -114,7 +117,7 @@ func TestAPI_ListGrants(t *testing.T) {
 	run := func(t *testing.T, tc testCase) {
 		req := httptest.NewRequest(http.MethodGet, tc.urlPath, nil)
 		req.Header.Set("Authorization", "Bearer "+accessKey)
-		req.Header.Add("Infra-Version", "0.18.2")
+		req.Header.Add("Infra-Version", "0.21.0")
```

Comment:
If this should be testing against the latest API, it should use `apiVersionLatest` instead